Common/yaml/Contacts.yaml

```diff
@@ -0,0 +1,374 @@
+openapi: 3.0.3
+info:
+  title: Smart Contact Mangaer OpenAPI 3.0
+  description: The Project was about creating smart contact manager.Which can be specifically used for business model.This is a web application which can be used as a simple pluging to any existing project.|-
```

Comment:
It should be "The project is*"
Correct the spell of plugin
Remove the pipe and hyphen symbol at last

Reply:
i have fixed.please review again.

Reply:
i have fixed.please review again.